src/components/SocialComments/styled.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components'
+import media from 'styled-media-query'
+
+export const Wrapper = styled.section`
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 5rem 1rem;
+`
+
+export const Title = styled.h1`
+  font-weight: 300;
+  font-size: min(4rem, 6vw);
+  text-align: center;
+`
+
+export const Grid = styled.div`
+  column-count: 3;
+  column-gap: 20px;
+  margin-top: 4.8rem;
+  padding: 0 1rem;
+
+  ${media.lessThan('large')`
+    column-count: 2;
+  `}
+
+  ${media.lessThan('small')`
+    column-count: 1;
+  `}
+`
+export const Box = styled.div`
+  background-color: var(--blue-300);
+  border-radius: 4px;
+
+  padding: 24px 16px;
+  margin-bottom: 20px;
+  font-size: 1rem;
+
+  break-inside: avoid;
+`
+
+export const Comment = styled.p`
```

Comment:
Ao invés de `p`, você poderia usar estrutura de `blockquote` que é exatamente para esses casos.

https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/